submissions/Oleksandr-Heleta/popup/index.html

```diff
@@ -0,0 +1,170 @@
+                                    <img class="popup_img" src="./popup-icons/mail.png" alt="mail">
+                                    <p>Mail</p>
+                                </a>
+                            </li>
+                            <li class="popup_item">
+                                <a class="popup_link" href="#" target="_blank" rel="noopener noreferrer">
+                                    <img class="popup_img" src="./popup-icons/news.png" alt="news">
+                                    <p>News</p>
+                                </a>
+                            </li>
+                            <li class="popup_item">
+                                <a class="popup_link" href="#" target="_blank" rel="noopener noreferrer">
+                                    <img class="popup_img" src="./popup-icons/maps.png" alt="map">
+                                    <p>Map</p>
+                                </a>
+                            </li>
+                            <li class="popup_item">
+                                <a class="popup_link" href="#" target="_blank" rel="noopener noreferrer">
+                                    <img class="popup_img" src="./popup-icons/files.png" alt="files">
+                                    <p>Failes</p>
+                                </a>
+                            </li>
+                            <li class="popup_item">
+                                <a class="popup_link" href="#" target="_blank" rel="noopener noreferrer">
+                                    <img class="popup_img" src="./popup-icons/contacts.png" alt="contacts">
+                                    <p>Contacts</p>
+                                </a>
+                            </li>
+                            <li class="popup_item">
+                                <a class="popup_link" href="#" target="_blank" rel="noopener noreferrer">
+                                    <img class="popup_img" src="./popup-icons/photos.png" alt="photos">
+                                    <p>Photos</p>
+                                </a>
+                            </li>
+                            <li class="popup_item">
+                                <a class="popup_link" href="#" target="_blank" rel="noopener noreferrer">
+                                    <img class="popup_img" src="./popup-icons/apps.png" alt="apps">
+                                    <p>Apps</p>
+                                </a>
+                            </li>
+                            <li class="popup_item">
+                                <a class="popup_link" href="#" target="_blank" rel="noopener noreferrer">
+                                    <img class="popup_img" src="./popup-icons/tanslation.png" alt="tanslation">
+                                    <p>Translation</p>
+                                </a>
+                            </li>
```

Comment:
Check and fix styles on focus: add straight borders
![image](https://user-images.githubusercontent.com/16196199/185042919-bb46d7e7-5992-4250-a161-6f7bdfb84f03.png)